src/App.scss

```diff
@@ -1,6 +1,11 @@
 @import "~@edx/edx-bootstrap/sass/edx/theme";
 @import "~bootstrap/scss/bootstrap";
 
+//TODO: colors should be defined in base.scss
+:root {
+  --dark-grey: #292b2c;
```

Comment:
We can't use CSS variables because they aren't supported in IE11: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables#compat-desktop